src/sql/orderbook/order_rewards.sql

```diff
@@ -31,6 +32,6 @@ select concat('0x', encode(trade_hashes.order_uid, 'hex')) as order_uid,
            when reward is null and fee_amount = 0 then False
            end                              as safe_liquidity
 from trade_hashes
-         left outer join {{reward_table}} o
+         left outer join order_execution o
```

Comment:
Instead of removing this unrelated change here, I picked up the remaining remnants of this hack fix and removed them here as well. Glad you noticed this was unrelated, but its needs to be taken out and here is a reasonable place to do so.